service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceMethod.java

```diff
@@ -188,40 +266,35 @@ private synchronized MethodResult invokeMethod(URL url, String methodName, Long
      * @throws IotHubException if the http request failed.
      */
     public Job scheduleDeviceMethod(String queryCondition,
```

Comment:
how about where parameters show up when put on newlines? First param is in line or new line?

Reply:
Should be on new line, so I'll fix this